libraries/rib-base/src/main/java/com/badoo/ribs/minimal/reactive/CompositeCancellable.kt

```diff
@@ -0,0 +1,13 @@
+package com.badoo.ribs.minimal.reactive
+
+class CompositeCancellable(vararg cancellables: Cancellable) : Cancellable {
+    private val items = mutableListOf(*cancellables)
+    operator fun plusAssign(item: Cancellable) {
+        items += item
```

Comment:
Should we do something, when CompositeCancellable is already cancelled and we are trying to add new Cancellable to the list?
CompositeDisposable disposes automatically in this case AFAIK.

Reply:
For me this is beyond the scope of this PR (can lead down a longer road), but feel free to open another one if you're interested.